### PR TITLE
6113_GetAPI

### PR DESCRIPTION
**Pull request by dinhquang251**
Get data from API using retrofit
https://edu-redmine.framgia.vn/issues/6113

### CODE REVIEW

**Comment by dinhquang251** [comment]
ready

File: app/src/main/java/com/framgia/moviedb_35/data/source/remote/MovieRemoteDataSource.java [hunk 1]
```diff
@@ -1,4 +1,40 @@
...
+    }
+
+    @Override
+    public Single<List<Movie>> getPopular(int page) {
+        return mRetrofitRequest.getPopular(page).map(new Function<CategoryData, List<Movie>>() {
+            @Override
+            public List<Movie> apply(CategoryData categoryData) throws Exception {
+                return categoryData.getMovies();
+            }
+        });//operator
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/data/source/remote/MovieRemoteDataSource.java]: remove comment

File: app/src/main/java/com/framgia/moviedb_35/data/source/remote/service/SettingRetrofitClient.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    private static RetrofitRequest sApiInstance;
+
+    public static void initialize(Application application) {
+        sApiInstance = createRetrofit(application, END_POINT_URL, RetrofitRequest.class);
+    }
+
+    public static RetrofitRequest getInstance() {
+        if (sApiInstance == null) {
+            throw new IllegalStateException(SettingRetrofitClient.class.getSimpleName()
+                    + EXCEPTION_MESSAGE);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/moviedb_35/data/source/remote/service/SettingRetrofitClient.java]: not + String

**Comment by dinhquang251** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Comment by dinhquang251** [comment]
ready

**Pull request merged**